sdk/src/Core/Amazon.Runtime/Internal/Util/MultiValueHeaderParser.cs

```diff
@@ -51,12 +51,12 @@ public static List<string> ToStringList(string header)
 
             int startAtIndex = 0;
             var bytes = System.Text.Encoding.UTF8.GetBytes(header);
-            Tuple<string, int> readResponse;
```

Comment:
There's another place where we need to use a `Tuple` in 3.5: https://github.com/aws/aws-sdk-net/blob/fab713ecd779581b23a46f4b6c500934e1744b9d/sdk/src/Core/Amazon.Runtime/Internal/Util/IniFile.cs#L50-L61

Would it make sense to follow the same pattern here? Then when `BCL35` is removed in `v4` we'll use the `Tuple` type from the BCL.

Reply:
I will make this change as it does make since to automatically drop an extra class when we drop BCL35 even if it is all private now.

Reply:
I also think having the Tuple class like above would be good. That way the resulting code can stay the same when we drop BCL35.